src/SimplePie.php

```diff
@@ -407,6 +407,11 @@ class SimplePie
      */
     public const FILE_SOURCE_FILE_GET_CONTENTS = 16;
 
+    /**
+     * @internal Default HTTP Accept header when sending HTTP requests
```

Comment:
Maybe something like:

```suggestion
     * @internal Default value of the HTTP Accept header when fetching/locating feeds
```

We probably will not want it for e.g. downloading and caching images. And header would consist of the `Accept: ` prefix so mention it is a value.